src/client/Client.js

```diff
@@ -227,6 +227,15 @@ class Client extends BaseClient {
     }
   }
 
+  /**
+   * Returns whether the client has logged in, indicative of being able to access
+   * properties such as `user` and `application`.
+   * @returns {boolean}
+   */
+  isReady() {
+    return this.ws.status === 0;
```

Comment:
isn't there an enum for this ?

Reply:
Yes looks like it. I'll update the PR to use `Status.READY`.